SECURITY_CONTACTS

```diff
@@ -12,7 +12,6 @@
 
 cjcullen
 joelsmith
-jonpulsifer
 liggitt
 philips
```

Comment:
philips has also been offboarded, and we added @micahhausler 

Mind syncing those changes here too?